infrastructure/parameters/dev.parameters.json

```diff
@@ -59,6 +59,9 @@
     "publicApiUrl": {
       "value": "dev.statistics.api.education.gov.uk"
     },
+    "publicApiDocsUrl": {
```

Comment:
Do you envision this changing per environment?  This could be a default value which can be overridden if necessary.

Reply:
I think we're likely to just have 2 environments for the docs site - dev and prod. 

Dev would be the GitHub hosted page and prod will be an Azure static site (when we get around to it).

I'll go with your suggestion of setting the GitHub page as the default anyway.